pyproject.toml

```diff
@@ -107,6 +107,8 @@ mkdocs-include-markdown-plugin = "~3.0.1"
 mkdocs = "~1.1.2"
 # Integration Tests
 requests = "~2.25.1"
+# Selenium web drivers for live integration testing
+selenium = "=3.141.0"
```

Comment:
Can this be a `~` requirement?

Reply:
It was initially but there are two issues:

1. For whatever reason, `poetry install` kept failing w/ the tilde requirement. I would like to figure this out.
2. There hasn't been an update to Selenium 3 since 2018. v4-beta has been in the works for several years.